.prow.yaml

```diff
@@ -292,7 +292,7 @@ presubmits:
   # e2e tests for Kubernetes 1.15
   #########################################################
 
-  - name: pre-kubermatic-e2e-aws-coreos-1.15
```

Comment:
We switched to `flatcar` in 2.14 and 2.15, maybe we should do the same for consistency.

Reply:
well, probably in v2.13 was not supported yet

Reply:
Yep, I confirm.